cmd/egnaddrs/main_test.go

```diff
@@ -8,11 +8,7 @@ import (
 )
 
 const (
-	// just copied this file from an eigencert deployment and hardcoded addrs below
-	// TODO(samlaf): eventually we should make updating this file automated
-	anvilStateFileName      = "eigenlayer-and-registries-deployed-anvil-state.json"
-	serviceManagerAddr      = "0x7a2088a1bFc9d81c55368AE168C2C02570cB814F"
-	registryCoordinatorAddr = "0x09635F643e140090A9A8Dcd712eD6285858ceBef"
```

Comment:
addresses are no longer hardcoded. We get them from the ContractsRegistry that is deployed as part of the integration test deployment.